src/main/native/include/support/jni_util.h

```diff
@@ -59,7 +60,7 @@ class JClass {
     env->DeleteLocalRef(local);
   }
 
-  void free(JNIEnv *env) {
+  void std::free(JNIEnv* env) {
```

Comment:
Incorrect addition of "std::".  I'm surprised this actually compiles..

Reply:
I think this is left over from before I fixed the bug in wpiformat.